hack/lib.sh

```diff
@@ -36,6 +38,11 @@ function hack::verify_helm() {
     if test -x "$HELM_BIN"; then
         local v=$($HELM_BIN version --short --client | grep -o -P '\d+.\d+.\d+')
         [[ "$v" == "$HELM_VERSION" ]]
+        return
+		fi
+		if test -x "$HELM2_BIN"; then
+        local v=$($HELM2_BIN version --short --client | grep -o -P '\d+.\d+.\d+')
+        [[ "$v" == "$HELM2_VERSION" ]]
```

Comment:
it's not right to check helm2 and helm3 in one function. if `$HELM_BIN` exist, `$HELM2_BIN` will not be checed.

you can add another two functions for `helm2`, e.g. hack::verify_helm2, hack::install_helm2.

Reply:
Done with new commit